project/BinaryIncompatibilities.scala

```diff
@@ -6,6 +6,40 @@ object BinaryIncompatibilities {
   )
 
   val Tools = Seq(
+      ProblemFilters.exclude[MissingMethodProblem](
+        "org.scalajs.core.tools.linker.backend.emitter.ScalaJSClassEmitter.this"),
```

Comment:
Thing is most of the ScalaJSClassEmitter are neither private nor private[emitter],
Same goes for both its constructors! I'm not sure what a proper message would then be